benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml

```diff
@@ -8,7 +8,6 @@ template_data:
   shared:
     pin_node1: {{ pin_node1 }}
     pin_node2: {{ pin_node2 }}
-    transactions: 100000
```

Comment:
You should be able to leave this here, and override it only for postgres.  You can also put the rampup time here.

Reply:
Fixed